src/dataLoaderService.ts

```diff
@@ -143,4 +145,22 @@ export class DataLoaderService {
       cacheKeyFn: ({ userId }) => defaultCacheKeyFn({ userId }),
     });
   }
+
+  get user() {
+    return this.getLoader<{ userId: string }, User>({
+      type: 'user',
+      loadFn: async ({ userId }) => {
+        if (!userId) {
+          return null;
+        }
+
+        return queryReadReplica(this.ctx.con, async ({ queryRunner }) => {
+          return queryRunner.manager
+            .getRepository(User)
+            .findOneBy({ id: userId });
```

Comment:
should we load specific fields? i feel like querying the readme and some other long columns can be wasteful

Reply:
done 2a9b6c07248f26d25bb95342ba7b7307687ae31a